cli/pcluster/easyconfig.py

```diff
@@ -11,25 +11,42 @@
 # fmt: off
 from __future__ import absolute_import, print_function  # isort:skip
 from future import standard_library  # isort:skip
+
 standard_library.install_aliases()
 # fmt: on
 
+import copy
 import errno
 import functools
 import logging
 import os
 import stat
 import sys
+import tempfile
 from builtins import input
 
 import boto3
 import configparser
 from botocore.exceptions import BotoCoreError, ClientError
 
+from pcluster.utils import get_supported_os, get_supported_schedulers
+
 from . import cfnconfig
 
 logger = logging.getLogger("pcluster.pcluster")
 unsupported_regions = ["ap-northeast-3"]
+DEFAULT_VALUES = {
+    "cluster_template": "default",
+    "aws_region_name": "us-east-1",
+    "scheduler": "sge",
+    "os": "alinux",
+    "max_queue_size": "10",
+    "master_instance_type": "t2.micro",
+    "compute_instance_type": "t2.micro",
+    "vpc_name": "public",
+    "initial_cpus": "1",
```

Comment:
I'd call `initial_size` because it could be  `initial_queue_size ` or  `desired_vcpus `